tests/framework/PostProcessors/BasicStatistics/factorial_analytic.xml

```diff
@@ -143,6 +171,96 @@
       <Input>x,y,z</Input>
       <Output>x1</Output>
     </PointSet>
+    <PointSet name="analyticalTest_basicStatPP">
+      <Output>analyticalTest_vars</Output>
+    </PointSet>
+    <PointSet name="analyticalTest_basicStatPP_pb">
+      <Output>analyticalTest_vars</Output>
+    </PointSet>
   </DataObjects>
 
+  <VariableGroups>
+    <Group name="analyticalTest_vars">skew_x,
+                 skew_y,
+                 skew_z,
+                 vc_x,
+                 vc_y,
+                 vc_z,
+                 percentile_5_x,
+                 percentile_5_y,
+                 percentile_5_z,
+                 percentile_95_x,
+                 percentile_95_y,
+                 percentile_95_z,
+                 mean_x,
+                 mean_y,
+                 mean_z,
+                 kurt_x,
+                 kurt_y,
+                 kurt_z,
+                 median_x,
+                 median_y,
+                 median_z,
+                 max_x,
+                 max_y,
+                 max_z,
+                 min_x,
+                 min_y,
+                 min_z,
+                 samp_x,
+                 samp_y,
+                 samp_z,
+                 var_x,
+                 var_y,
+                 var_z,
+                 sigma_x,
+                 sigma_y,
+                 sigma_z,
+                 nsen_x_x,
+                 nsen_x_y,
+                 nsen_x_z,
+                 nsen_y_x,
+                 nsen_y_y,
+                 nsen_y_z,
+                 nsen_z_x,
+                 nsen_z_y,
+                 nsen_z_z,
+                 sen_x_x,
+                 sen_x_y,
+                 sen_x_z,
+                 sen_y_x,
+                 sen_y_y,
+                 sen_y_z,
+                 sen_z_x,
+                 sen_z_y,
+                 sen_z_z,
+                 pear_x_x,
+                 pear_x_y,
+                 pear_x_z,
+                 pear_y_x,
+                 pear_y_y,
+                 pear_y_z,
+                 pear_z_x,
+                 pear_z_y,
+                 pear_z_z,
+                 cov_x_x,
+                 cov_x_y,
+                 cov_x_z,
+                 cov_y_x,
+                 cov_y_y,
+                 cov_y_z,
+                 cov_z_x,
+                 cov_z_y,
+                 cov_z_z,
+                 vsen_x_x,
+                 vsen_x_y,
+                 vsen_x_z,
+                 vsen_y_x,
+                 vsen_y_y,
+                 vsen_y_z,
+                 vsen_z_x,
+                 vsen_z_y,
+                 vsen_z_z</Group>
```

Comment:
maybe it's not worth changing, but we should create these groups without the newlines, as it clogs up the input file.

Reply:
Not sure, it is easy for me to check the variables.